src/sentry/static/sentry/less/group-detail.less

```diff
@@ -1606,7 +1538,7 @@ ul.crumbs {
           display: block;
           width: 100%;
           height: 15px;
-          #gradient > .vertical(rgba(255, 255, 255, 0), rgba(255, 255, 255, 1));
+          #gradient > .vertical(rgba(255, 255, 255, 0), rgba(255, 255, 255, 1));;
```

Comment:
```suggestion
          #gradient > .vertical(rgba(255, 255, 255, 0), rgba(255, 255, 255, 1));
```

Reply:
I noticed this, but the automatic formatter always added the ;

Reply:
Wacky formatter is being wacky.